shared/src/main/scala/com/raquo/domtypes/generic/defs/attrs/svgAttrs.scala

```diff
@@ -0,0 +1,45 @@
+
+package com.raquo.domtypes.generic.defs.attrs
+
+import com.raquo.domtypes.generic.builders.AttrBuilder
+import com.raquo.domtypes.generic.codecs.{BooleanAsOnOffStringCodec, BooleanAsTrueFalseStringCodec}
+
+/** @tparam A HTML Attribute, canonically [[com.raquo.domtypes.generic.keys.Attr]] */
```

Comment:
Comment no longer accurate – SVG, not HTML.

Also, we should not reuse the `Attr` key for this because you can't use SVG attrs with HTML elements. We need a new SvgAttr key.

Reply:
Fixed,created new keys/SvgAttr.scala